index.js

```diff
@@ -11,40 +11,37 @@ function instrument(options) {
   var instrumenter = new Instrumenter.Instrumenter();
   var captured = false;
 
-  function transform(file) {
-    // If if doesnt match the pattern dont instrument it
+  return through.obj(function(row, enc, next) {
+    if (row.file.indexOf("node_modules") !== -1) {
```

Comment:
I didn't explain myself correctly. I was saying that this use case can be handled in line 10 by adding a pattern by default. In that way, there's a way to override it and allow to instrument files under `node_modules`.